packages/@unimodules/core/ios/UMCore/UMAppDelegateWrapper.h

```diff
@@ -5,6 +5,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface UMAppDelegateWrapper : UIResponder <UIApplicationDelegate>
 
+@property (nullable, nonatomic, strong) UIWindow *window;
```

Comment:
I have mixed feelings about this.

On one hand this property is declared `@optional` in `UIApplicationDelegate`, so adding it to `AppDelegate` or its superclass shouldn't be a problem.

On the other hand in a regular Swift project one would add `var window: UIWindow?` to the app's `AppDelegate`, but that, in our case, results in a compile error: `Cannot override with a stored property 'window'`.

Another way I found to fix this problem was adding `@synthesize window = _window;` to `UMAppDelegateWrapper.m`. This would tell the compiler to create getters and setters for the `window` property, which is basically what would fix the `unrecognized selector setWindow: sent…`. This would use the property memory added by the system (?) while adding a `@property` here would create another stored property, similarly to what Swift compiler complains about when adding `var window` to `AppDelegate` (?).

I feel like adding the `@synthesize` would be somehow more scoped and more safe, but let me also ask @tsapeta for a second opinion.

Reply:
I agree with @sjchmiela, `@synthesize` seems to be safer here 👍